### PR TITLE
[DOC release] Replace docs for render Fixes #12168

### LINKED ISSUE #12168: {{render}} docs lost during 1.13 rewrite
See original docs in https://github.com/emberjs/ember.js/blob/v1.12.1/packages/ember-routing-htmlbars/lib/helpers/render.js#L19

Most of those docs should be added to https://github.com/emberjs/ember.js/blob/master/packages/ember-routing-htmlbars/lib/keywords/render.js.

### PR DESCRIPTION
**Pull request by mwils**
lost during 1.13 rewrite

### CODE REVIEW

**Comment by mwils** [comment]
Let me know if something needs changing.

File: packages/ember-routing-htmlbars/lib/keywords/render.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/**
+@module ember
+@submodule ember-routing-htmlbars
```
Comment (rwjblue) [inline packages/ember-routing-htmlbars/lib/keywords/render.js]: Can you tweak this to `ember-templates`?

File: packages/ember-routing-htmlbars/lib/keywords/render.js [hunk 2]
```diff
@@ -8,6 +13,63 @@ import generateController from 'ember-routing/system/generate_controller';
...
+  If a view class with the same name exists, the view class also will be used.
+  Note: A given controller may only be used *once* in your app in this manner.
+  A singleton instance of the controller will be created for you.
+  Example:
+  ```javascript
+  App.NavigationController = Ember.Controller.extend({
+    who: "world"
+  });
+  ```
+  ```handlebars
```
Comment (rwjblue) [inline packages/ember-routing-htmlbars/lib/keywords/render.js]: Should be a blank line between example blocks.

File: packages/ember-routing-htmlbars/lib/keywords/render.js [hunk 3]
```diff
@@ -8,6 +13,63 @@ import generateController from 'ember-routing/system/generate_controller';
...
+  <h1>My great app</h1>
+  {{render "navigation"}}
+  ```
+  ```html
+  <h1>My great app</h1>
+  <div class='ember-view'>
+    Hello, world.
+  </div>
+  ```
+  Optionally you may provide a second argument: a property path
```
Comment (rwjblue) [inline packages/ember-routing-htmlbars/lib/keywords/render.js]: Needs a blank line after the code fence above.

File: packages/ember-routing-htmlbars/lib/keywords/render.js [hunk 4]
```diff
@@ -8,6 +13,63 @@ import generateController from 'ember-routing/system/generate_controller';
...
+  ```javascript
+  App.NavigationController = Ember.Controller.extend({
+    who: "world"
+  });
+  ```
+  ```handlebars
+  <!-- navigation.hbs -->
+  Hello, {{who}}.
+  ```
+  ```handlebars
```
Comment (rwjblue) [inline packages/ember-routing-htmlbars/lib/keywords/render.js]: Needs a blank line after the code fence above.

File: packages/ember-routing-htmlbars/lib/keywords/render.js [hunk 5]
```diff
@@ -8,6 +13,63 @@ import generateController from 'ember-routing/system/generate_controller';
...
+  <!-- application.hbs -->
+  <h1>My great app</h1>
+  {{render "navigation"}}
+  ```
+  ```html
```
Comment (rwjblue) [inline packages/ember-routing-htmlbars/lib/keywords/render.js]: Needs a blank line after the code fence above.

File: packages/ember-routing-htmlbars/lib/keywords/render.js [hunk 6]
```diff
@@ -8,6 +13,63 @@ import generateController from 'ember-routing/system/generate_controller';
...
+    Hello, world.
+  </div>
+  ```
+  Optionally you may provide a second argument: a property path
+  that will be bound to the `model` property of the controller.
+  If a `model` property path is specified, then a new instance of the
+  controller will be created and `{{render}}` can be used multiple times
+  with the same name.
+ For example if you had this `author` template.
+ ```handlebars
```
Comment (rwjblue) [inline packages/ember-routing-htmlbars/lib/keywords/render.js]: Needs a blank line after the code fence above.

File: packages/ember-routing-htmlbars/lib/keywords/render.js [hunk 7]
```diff
@@ -8,6 +13,63 @@ import generateController from 'ember-routing/system/generate_controller';
...
+You could render it inside the `post` template using the `render` helper.
+```handlebars
+<div class="post">
+<h1>{{title}}</h1>
+<div>{{body}}</div>
+{{render "author" author}}
+</div>
+ ```
+  @method render
+  @for Ember.Handlebars.helpers
```
Comment (rwjblue) [inline packages/ember-routing-htmlbars/lib/keywords/render.js]: `@for Ember.Templates.helpers`

**Comment by rwjblue** [comment]
Awesome, thank you!  I left some inline tweaks...

**Comment by mwils** [comment]
My pleasure. I will make the mods and commit.

**Comment by mwils** [comment]
Please check it over and let me know if I need to make any other tweaks.

**Comment by mwils** [comment]
And should i merge/flatten my commits?

**Comment by rwjblue** [comment]
Yes, please

**Comment by rwjblue** [comment]
Awesome, thank you!

**Pull request merged**